.travis.yml

```diff
@@ -44,7 +45,7 @@ install:
   - stty cols 120 && composer show
 
 script:
-  - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then XDEBUG_MODE=coverage composer test-coverage ; else composer test ; fi
```

Comment:
Does this give any benefit?
There are 0 laminas components which added this and thus I think we can omit it here.

Reply:
I'm not to sure where that came from, but it works without it, so I removed it.